ee/clickhouse/views/test/test_clickhouse_retention.py

```diff
@@ -64,6 +65,66 @@ def test_retention_test_account_filters(self):
             "Day 1": {"1": ["person 3"]},
         }
 
+    @snapshot_clickhouse_queries
+    def test_retention_aggregation_by_distinct_id_and_retrieve_people(self):
+        organization = create_organization(name="test")
+        team = create_team(organization=organization)
+        user = create_user(email="test@posthog.com", password="1234", organization=organization)
+        config.AGGREGATE_BY_DISTINCT_IDS_TEAMS = f"{team.pk}"
```

Comment:
Is this reset on test teardown?

Can we instead use the pytest django settings fixture which should handle this.

It would be good to keep things simple in the tests and not dependent on the constance interface if possible, but just the django settings interface.

Reply:
great shout, did this!